ts/src/base/Exchange.ts

```diff
@@ -6221,6 +6221,74 @@ export default class Exchange {
     parseLeverage (leverage, market: Market = undefined): Leverage {
         throw new NotSupported (this.id + ' parseLeverage() is not supported yet');
     }
+
+    convertExpireDate (date: string): string {
+        // parse YYMMDD to datetime string
+        const year = date.slice (0, 2);
+        const month = date.slice (2, 4);
+        const day = date.slice (4, 6);
+        const reconstructedDate = '20' + year + '-' + month + '-' + day + 'T00:00:00Z';
+        return reconstructedDate;
+    }
+
+    convertExpireDateToMarketIdDate (date: string): string {
```

Comment:
also, right now deribit/bybit seem to share the same marketId logic but not sure if we can assume that as the "generic" way because it will depend from exchange to Exchange. We can also rename this to something like parseDDMMYYToDDMonthYear (yeah a bit weird)

Reply:
> also, right now deribit/bybit seem to share the same marketId logic but not sure if we can assume that as the "generic" way because it will depend from exchange to Exchange.

We can always override it if needed, and then it will be more clear that the overridden method is different. Right now we have the same code on multiple exchanges